clang/CMakeLists.txt

```diff
@@ -237,7 +237,8 @@ set(ENABLE_X86_RELAX_RELOCATIONS OFF CACHE BOOL
 set(ENABLE_EXPERIMENTAL_NEW_PASS_MANAGER FALSE CACHE BOOL
   "Enable the experimental new pass manager by default.")
 
-set(CLANG_SPAWN_CC1 OFF CACHE BOOL
+# [SYCL] workaround for memory consumption issue
```

Comment:
Please, clarify what is the root cause of the issue.

Reply:
Clang tool executes cc1 commands in the same process after https://github.com/intel/llvm/commit/b4a99a061f517e60985667e39519f60186cbb469#diff-c01c461d91e3f7c612a62f5bddd23d41

It causes increasing memory consumption for compilations where several source files are passed (for C++ application) or several build steps (e.g. for SYCL application we have host, device and integration header step per source file). Due to some resone memory is not freed for all cc1 steps until end of overall process.

This WA revert back to previous behavior (separate process for every CC1 command)

Reply:
@bader i am merging the pull down and will followup with investigation and possible fix next week.